src/hooks.ts

```diff
@@ -199,16 +200,15 @@ interface AxisId {
 }
 export function useMaxMin(options: AxisId = {}) {
   const { xAxis = 'x', yAxis = 'y' } = options;
-  const { axisContext, plotWidth, plotHeight } = usePlotContext();
-  const [xScale, yScale] = validateAxis(axisContext, xAxis, yAxis);
+  const overrides = usePlotOverrides();
   return {
     [xAxis]: {
-      max: toNumber(xScale?.invert(plotWidth)),
-      min: toNumber(xScale?.invert(0)),
+      max: overrides[xAxis]?.max,
+      min: overrides[xAxis]?.min,
     },
     [yAxis]: {
-      max: toNumber(yScale?.invert(0)),
-      min: toNumber(yScale?.invert(plotHeight)),
+      max: overrides[yAxis]?.max,
+      min: overrides[yAxis]?.min,
     },
   };
 }
```

Comment:
@stropitek could you please help me , why `useMaxMin` is not working

Reply:
the two versions are not working (when i use scale.invert and also when i use overrides)

Reply:
I would suggest that you start by implementing the pan feature without the alt key and without other hooks for zooming and scrolling. That pan feature should use the existing `useStartMoveEnd` hook